docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,27 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes filter versions that will not run on JDK 8.
+This issue is fixed in the latest versions: 
+
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v3.1.12-plugins-filters-date.html[date filter 3.1.12]
+* https://www.elastic.co/guide/en/logstash-versioned-plugins/current/v7.2.7-plugins-filters-geoip.html[geoip filter 7.2.7]
```

Comment:
I couldn't use our usual link formatting here because 7.16.1 will lock the plugins (and plugin docs) to the shipped plugin versions. I linked to the updated plugins in the Versioned Plugin Reference to be more precise and to avoid confusing people.